package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "^8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "^0.6.0-beta",
+    "@tacc/core-styles": "^0.6.0-beta.2",
```

Comment:
The relevant change is [TACC/tup-ui#19:`/libs/core-styles/src/lib/_imports/objects/o-section.css`](https://github.com/TACC/tup-ui/pull/19/files#diff-8170f5f1432376a5a03f0139f211a030bd8e3eb9320f643410227b087d5080e7).